dangerzone/cli.py

```diff
@@ -38,6 +38,7 @@ def print_header(s: str) -> None:
     type=click.UNPROCESSED,
     callback=args.validate_input_filenames,
 )
+@click.version_option(version=get_version())
```

Comment:
Suggestion: if we add `... , message="%(version)s)`, we don't have to see the prog name in the output, nor do any special parsing.

Other than that, the PR looks good.

Reply:
I missed this one. Actually, we don't even need to specify anything. `@click.version_option()` works out of the box. And it's function is a bit smarter than our `get_version()`.

It would be nice if we could use click's method for determining the version in runtime instead of having `share/version.txt`. But for now I'd say we can leave that since many things depend on that method (`setup.py`, `build-{deb,rpm}.py, etc.)

So I think I'll go ahead and leave it as it was (since you approved the PR already). This way getting the version from `share/version.txt` is the canonical place.

Is that OK?

Reply:
Yeah, my comment was not directed to `get_version()`, but making `--version` print just the version number, instead of the program name as well, which makes it difficult to parse (I'm referring to the code in the `tests/`)

Anw, that's just a suggestion. Feel free to merge it as is.

Reply:
Ah. Then I agree. I have added `... , message="%(version)s)` and simplified the test `tests/test_cli.py::TestCliBasic::test_version`.

Reply:
Nice, feel free to merge.